src/sharepoint/information/INFORMATION.html

```diff
@@ -6,83 +6,69 @@
-                                <li class="oui-tile__item">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block" data-translate="sharepoint_dashboard_url_label"></strong>
-                                        <a data-ng-href="https://{{informationsCtrl.sharepoint.url}}" target="_blank"
-                                           data-ng-if="informationsCtrl.sharepoint.state !== 'creating'">
-                                            <span data-ng-bind="'https://' + informationsCtrl.sharepoint.url"></span>
-                                            <span class="fa fa-external-link" aria-hidden="true"></span>
-                                        </a>
-                                        <span data-translate="sharepoint_dashboard_creating_url_label"
-                                              data-ng-show="informationsCtrl.sharepoint.state === 'creating'"></span>
-                                    </div>
-                                </li>
-                                <li class="oui-tile__item"
-                                    data-ng-if="informationsCtrl.associatedExchange">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block" data-translate="sharepoint_dashboard_linked_exchange_label"></strong>
-                                        <span data-ng-bind="SharepointCtrl.exchangeId"
-                                              data-ng-if="!informationsCtrl.associatedExchangeLink"></span>
-                                        <a data-ng-href="{{informationsCtrl.associatedExchangeLink}}"
-                                           data-ng-bind="SharepointCtrl.exchangeId"
-                                           data-ng-click="informationsCtrl.setExchange()"
-                                           data-ng-if="informationsCtrl.associatedExchangeLink">
-                                        </a>
-                                    </div>
-                                </li>
-                            </ul>
-                        </div>
-                    </div>
+                    <oui-tile class="h-100" data-heading="{{:: 'sharepoint_dashboard_access' | translate}}">
+                        <oui-tile-definition data-term="{{:: 'sharepoint_dashboard_url_label' | translate}}">
+                            <oui-tile-description>
+                                <a data-ng-href="https://{{informationsCtrl.sharepoint.url}}" 
+                                   target="_blank"
+                                   data-ng-if="informationsCtrl.sharepoint.state !== 'creating'">
+                                    <span data-ng-bind="'https://' + informationsCtrl.sharepoint.url"></span>
+                                    <span class="fa fa-external-link" aria-hidden="true"></span>
```

Comment:
Alternative text for screen-reader users

Reply:
```js
<span data-translate="common_open_new_tab" class="sr-only"></span>
```

Reply:
Fixed 😄